pkgs/by-name/re/readability-extractor/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "readability-extractor";
+  version = "0.0.10";
+
+  src = fetchFromGitHub {
+    owner = "ArchiveBox";
+    repo = pname;
+    rev = "be5c3222990d4f0459b21e74802565309bdd1d52";
```

Comment:
Can we open an upstream issue asking for tags?

Reply:
https://github.com/ArchiveBox/readability-extractor/issues/8